src/canopy-react-error-boundary.js

```diff
@@ -35,7 +35,7 @@ export default function decorateOptions(opts) {
           return (
             <div className="cps-modal" id="canopy-react-error-boundary-modal">
               <div className="cps-modal__screen" />
-              <div className="cps-modal__dialog cps-card__height-3">
+              <div className="cps-modal__dialog cps-card__height-3" style={{maxWidth: '100vw', transform: "translateX(-50%)", left: "50%"}}>
```

Comment:
this makes it so that it's never bigger than the width of the screen, and also is always centered on both mobile and desktop devices.